pwiz_tools/Shared/PanoramaClient/PanoramaUtil.cs

```diff
@@ -608,17 +488,21 @@ public FolderState IsValidFolder(string folderPath, string username, string pass
                 {
                     return FolderState.notfound;
                 }
-                else throw;
+                else
+                {
+                    throw new PanoramaServerException(FolderState.unknown, folderPath, ex.Message, ServerUri,
+                        requestUri, Username);
+                }
             }
             return FolderState.valid;
         }
-    
-        public FolderOperationStatus CreateFolder(string folderPath, string folderName, string username, string password)
+
+        public FolderOperationStatus CreateFolder(string folderPath, string folderName)

```

Comment:
This is test support code for taking screenshots. Should be better commented and ideally moved to the AuditLogTutoriaTest.